utils/releaser/src/FilesProvider/DockerComposeFilesProvider.php

```diff
@@ -15,7 +15,7 @@ public function provide(): array
     {
         $finder = Finder::create()
             ->in(getcwd())
-            ->exclude('vendor')
+            ->exclude(['vendor', 'project-base/var/postgres-data', 'utils/releaser'])
```

Comment:
for mac there is also `./project-base/var/elasticsearch-data` and elasticsearch image is also executed via root like postgres but for windows elasticsearch-data is named volume so this is strange.

Reply:
for linux it is also named volume

Reply:
It makes sense to me to exclude the whole `project-base/var` folder...

> for linux it is also named volume

really? where is this defined :question: